### PR TITLE
Adding language support for Chinese i18n

### PR DESCRIPTION
**Pull request by eguo3**
initial i18n for Chinese, work in progress

### CODE REVIEW

File: openlibrary/i18n/zh/messages.po [hunk 1]
```diff
@@ -0,0 +1,782 @@
...
+msgstr "十分抱歉。您刚刚看的东西似乎出现了问题。"
+
+#: internalerror.html:15
+#, python-format
+msgid ""
+"We've noted the error %s and will look into it as soon as possible. Head for "
+"<a href=\"/\">home</a>?"
+msgstr ""
+"我们已注意到此错误 %s 并将尽快调查此错误。前往 "
+"<a href=\"/\">首页</a>?"
```
Comment (mekarpeles) [inline openlibrary/i18n/zh/messages.po:240]: Q: @cdrini there are a few cases where the msgstr is on a different line than the translations -- does this concatenate correctly?

**Comment by mekarpeles** [comment]
I checked several of the translation strings (with my limited 中文熟練 :sweat_smile: ) and they seem accurate @cdrini + @jimchamp 

@eguo3 thank you very much for your help! 謝謝妳幫了我們！

**Comment by mekarpeles** [comment]
Given the effects of this PR are Chinese translations, going to merge and we can patch if any of the translation strings need to be revised (e.g. the question I asked above).

Thank you again for your contribution!

**Comment by eguo3** [comment]
> I checked several of the translation strings (with my limited 中文熟練 😅 ) and they seem accurate @cdrini + @jimchamp
> 
> @eguo3 thank you very much for your help! 謝謝妳幫了我們！

Thank you for the proofreading!

**Comment by cdrini** [comment]
@mekarpeles please add `zh` to the `Makefile` i18n validation step; that'll run validation on it and make sure nothing causes errors

File: openlibrary/i18n/zh/messages.po (hunk 1 above)
Comment (cdrini) [reply openlibrary/i18n/zh/messages.po:240]: That should be fine; the validation step will confirm for certain though

**Pull request merged**